react-carousel-component/my-app/src/Indicators.tsx

```diff
@@ -0,0 +1,23 @@
+import { TiMediaRecord, TiMediaRecordOutline } from 'react-icons/ti';
+
+type Props = {
+  viewing: number;
+  images: { url: string; id: number }[];
+  onClick: (i: number) => void;
+};
+
+export function Indicators({ viewing, images, onClick }: Props) {
+  return images.map((element) => {
```

Comment:
The usual way dot indicators work is that all of them are hollow circles except the current one, which is solid.